compiler+runtime/test/cpp/jank/read/lex.cpp

```diff
@@ -4,6 +4,8 @@
 #include <jank/read/lex.hpp>
 
 /* This must go last; doctest and glog both define CHECK and family. */
+#include <iostream>
+#include <clang/Basic/Diagnostic.h>
```

Comment:
Please put these at the top of the file. Also, the comment on line 6 is specifically for the doctest include.

Reply:
we actually don't need them. removed.